### PR TITLE
Report hash verification progress

### LINKED ISSUE #210: Display integrity check progress
as of https://github.com/aria2/aria2/commit/2417225023d1bc94d994a639c7251ec6aa588f5f
,which should hopefully be in the next release of aria2, there should be new json keys in TellStatus namely `verifiedLength` and `verifiedPending`

It would be nice to use this new feature to display the progress of hashing which can take quite a while for large downloads. I noticed there is another outstanding issue about this #112 so I wasnt sure if I should comment there or make a new issue as its now possible.

I may submit a PR for this but unfortunately I dont have much time these days.

### PR DESCRIPTION
**Pull request by kkartaltepe**
Resolves #210 

This will show the progress of hash verification were previously the UI would only display the download as active with no progress until hash verification completed. I don't think this is a mergable state. I would still like to add in display that the remaining `active` but non-downloading items are queued. I'm just looking for feedback that this is appropriate and mergable or any changes you would like to see.

### CODE REVIEW

File: js/ctrls/main.js [hunk 1]
```diff
@@ -398,6 +398,10 @@ function(
 				animCollapsed: true,
 				files: [],
 			};
+			if (d.verifiedLength)
+				ctx.verifiedLength = d.verifiedLength;
```
Comment (ziahamza) [inline js/ctrls/main.js]: Will `verifiedLength` still exist after its all verified, and the real download starts? 

Normally my verifying time is low so cant test this for all edge cases ..

Comment (kkartaltepe) [inline js/ctrls/main.js]: No, verifiedLength only exists while the download is actively being verified. Similarly verifyIntegrityPending only exists while the download is in the queue to be verified.

Its easiest to notice on peicehashes like torrents since they take a bit longer to hash. Try creating a torrent for something like 10gb of files on your system and then checking integrity on them (Or grabbing your favorite 10gb torrent from somewhere).

**Comment by kkartaltepe** [comment]
This 2nd commit puts this at the features I wanted in my PR. I'm not sure if the `-warning` color scheme is best but I think it makes sense as its not yet downloading so it helps differentiate it from truly being `active`. Tested that the stop button shown during verifying works.

Let me know if there is a better way to implement this as I never really work with js or angular.

I'm testing this I just have a large torrent at 15GB and a handful of smaller ones cleverly named so that with `aria2c --enable-rpc --rpc-secret='xxx' -V *.torrent' this verifies the integrity of the the large torrent first and gives you ample time to see everything else queued up. (Be sure you have more than 1 download at a time allowed or the other downloads wont even queue for verification until the first download is completely finished)

**Comment by ziahamza** [comment]
@kkartaltepe I agree with showing it as a `warning` rather then `active`.

And your 2nd commit was the proper way of adding it in. The code looks good! Gonna release for everyone 🎉
